orc8r/cloud/go/obsidian/handler.go

```diff
@@ -322,3 +327,18 @@ func NetworkIdHttpErr() *echo.HTTPError {
 func TenantIdHttpErr() *echo.HTTPError {
 	return HttpError(fmt.Errorf("Missing Tenant ID"), http.StatusBadRequest)
 }
+
+// GetPaginationParams takes in context and returns page size and token params.
```

Comment:
Nit: "takes in a context" is redundant because it's clear from the function signature